File: src/main/java/checkdigitcli/command/CDCLI.java
package checkdigitcli.command;

import picocli.CommandLine.Command;

@Command( //
		name = "cdcli", //
		subcommands = {Exit.class, Help.class, GenerateCheckDigits.class} //
)
public class CDCLI implements Runnable {

	@Override
	public void run() {
		System.out.println("Use the command 'help' for help");
	}

}

File: src/main/java/checkdigitcli/command/CDCLI.kt
package checkdigitcli.command

import picocli.CommandLine

@CommandLine.Command(name = "cdcli", subcommands = [Exit::class, Help::class, GenerateCheckDigits::class])
class CDCLI : Runnable {
    override fun run() {
        println("Use the command 'help' for help")
    }
}